### ISSUE TITLE
C3_W4_Assignment.ipynb

### ISSUE THREAD

**Issue by naiborhujosua** [https-deeplearning-ai/tensorflow-1-public#46]
`seed_text = "Laurence went to dublin"
next_words = 100
  
for _ in range(next_words):
	token_list = tokenizer.texts_to_sequences([seed_text])[0]
	token_list = pad_sequences([token_list], maxlen=max_sequence_len-1, padding='pre')
	predicted = model.predict_classes(token_list, verbose=0)
	output_word = ""
	for word, index in tokenizer.word_index.items():
		if index == predicted:
			output_word = word
			break
	seed_text += " " + output_word
print(seed_text)`

predict_classes has been deprecated based on these answers [predict_classes](https://stackoverflow.com/questions/68836551/keras-attributeerror-sequential-object-has-no-attribute-predict-classes)

**Comment by cfav-dev** [comment]
Hi! Thank you for raising this. The labs have now been updated and the deprecated method has been removed. Thank you!
